src/node/rpc/test/tx_status_test.cpp

```diff
@@ -65,12 +65,12 @@ TEST_CASE("edge cases")
     INFO("Unknown views");
     // Impossible: view for all global txs must be known
     // get_tx_status(a, N, 0, b, >=N)
-    CHECK_THROWS(get_tx_status(3, 10, 0, 1, 10));
-    CHECK_THROWS(get_tx_status(3, 10, 0, 1, 11));
-    CHECK_THROWS(get_tx_status(3, 10, 0, 3, 10));
-    CHECK_THROWS(get_tx_status(3, 10, 0, 3, 11));
-    CHECK_THROWS(get_tx_status(3, 10, 0, 4, 10));
-    CHECK_THROWS(get_tx_status(3, 10, 0, 4, 11));
+    CHECK_THROWS(get_tx_status(3, 10, -1, 1, 10));
+    CHECK_THROWS(get_tx_status(3, 10, -1, 1, 11));
+    CHECK_THROWS(get_tx_status(3, 10, -1, 3, 10));
+    CHECK_THROWS(get_tx_status(3, 10, -1, 3, 11));
+    CHECK_THROWS(get_tx_status(3, 10, -1, 4, 10));
+    CHECK_THROWS(get_tx_status(3, 10, -1, 4, 11));
```

Comment:
I am not sure what the correct solution is here, will discuss and fix before pushing